relayer/chains/steller/listener.go

```diff
@@ -13,18 +13,41 @@ import (
 	relayertypes "github.com/icon-project/centralized-relay/relayer/types"
 	"github.com/icon-project/centralized-relay/utils/concurrency"
 	"github.com/icon-project/centralized-relay/utils/sorter"
-	xdr "github.com/stellar/go-xdr/xdr3"
+	xdr3 "github.com/stellar/go-xdr/xdr3"
 	"go.uber.org/zap"
 )
 
 func (p *Provider) Listener(ctx context.Context, lastSavedLedgerSeq uint64, blockInfo chan *relayertypes.BlockInfo) error {
+	if err := p.RestoreKeystore(ctx); err != nil {
+		return fmt.Errorf("failed to restore key: %w", err)
+	}
+	// p.QueryLastMessage()
+
+	go func() { //Todo remove: used temporarily for testing purpose only
```

Comment:
remove the commented block as it seems temporary as mentioned in comment itself.

Reply:
removed